api/spec/v1.yaml

```diff
@@ -149,6 +149,30 @@ paths:
                 $ref: '#/components/schemas/Status'
         <<: *common_error_responses
 
+  /consensus/total_supply_raw:
+    get:
+      summary: Returns the total supply of the consensus layer token.
+      responses:
+        '200':
+          description: A string containing the total supply of consensus layer token.
+          content:
+            text/plain:
```

Comment:
what's the story behind this, did someone want text/plain?

Reply:
Yeah the referenced issue

> Ideally the endpoint also supports raw output without json etc (for external services that will fetch this).

And since this is currently the only use case, I only implemented the "raw" text/plain endpoint.

Reply:
oh huh interesting